client-app/core/api/graphql/settings/queries/getStoreSettings/getStoreSettings.graphql

```diff
@@ -0,0 +1,36 @@
+#import "../../../fragments/allCurrencyFields.graphql"
+#import "../../../fragments/allLanguageFields.graphql"
+
+query GetStoreSettings($storeId: String!) {
+  store(storeId: $storeId) {
+    storeId
+    storeName
+    catalogId
+    storeUrl
+    userName
+    userId
```

Comment:
`userId` and `userName` is `id` and `userName` from `me` query and doesn't relate to store

Reply:
removed